src/resources/js/app.js

```diff
@@ -1,9 +1,10 @@
 require('./bootstrap');
 
-import Vue from 'vue';
+window.Vue = require('vue');
```

Comment:
Prefer the `import xxx from 'yyy';` over require.

See the [styleguide](https://github.com/airbnb/javascript#modules)

Reply:
Is ok now